### DISCUSSION TITLE
How do I copy the TailwindUI components WITH the alpine javascript included?

### DISCUSSION THREAD

**Discussion by jerodion** [tailwindlabs/tailwindcss#3849] [Help] [answered]
I want to be able to copy the TailwindUI components with the Alpine JS included. I don't want to have to view the page source and copy it from there.

Is there a way to do this?

**Comment by ghost** [comment]
This would be super helpful, I don't think they currently support it but it would definitely be appreciated.

**Comment by adamwathan** [comment, marked as answer]
Hey! See discussion here:

https://github.com/tailwindlabs/tailwindui-issues/issues/535

> Hey! We're planning to add proper support for Alpine in the future, and just starting with React/Vue because we have to start somewhere.
> 
> The reason we don't make the current Alpine demos easily available (other than copying from dev-tools) is that the code is very incomplete and not production ready. They are just quick proof-of-concept snippets to demo the different states, and don't include proper keyboard support, focus trapping, or other accessibility concerns. Some things behave in totally demo-specific ways as well, like the modal dialogs that automatically re-open after you close them.
> 
> First-class production-ready Alpine examples are definitely something we hope to tackle though!
